From 2287ae8adaa7f61024c3c38eecb10bf71bc9f08a Mon Sep 17 00:00:00 2001
From: Jean-Christophe Morin <jcmorin@anaconda.com>
Date: Thu, 23 Nov 2023 18:47:50 -0500
Subject: [PATCH] Only inlude sktime package.

This avoids having the build_tools, docs and other unnecessary things to be
installed in site-packages's root.

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index dd7fe360..94e28aa5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -302,7 +302,7 @@ sktime = [
 ]
 
 [tool.setuptools.packages.find]
-exclude = ["tests", "tests.*"]
+include = ["sktime*"]
 
 [tool.nbqa.exclude]
 black = "^docs/source/examples/"
-- 
2.42.1

